numpy/core/tests/test_scalarmath.py

```diff
@@ -667,9 +667,13 @@ def _test_abs_func(self, absfunc):
             x = tp(np.finfo(tp).min)
             assert_equal(absfunc(x), -x.real)
 
+    @pytest.mark.xfail(sys.platform == 'cygwin',
+                       reason='abs(np.complex256.max) overflows')
     def test_builtin_abs(self):
         self._test_abs_func(abs)
 
+    @pytest.mark.xfail(sys.platform == 'cygwin',
+                       reason='abs(np.complex256.max) overflows')
     def test_numpy_abs(self):
         self._test_abs_func(np.abs)
```

Comment:
Rather than doing this, it would be good to make `_test_abs_func` use `pytest.mark.parametrize` so that we can only xfail `complex256`

Reply:
Done.  It would be nice to have something similar for the branch cut tests, but there's too much variation to make that simple.

Reply:
Now done in a way that doesn't crash if the platform doesn't have a 256-bit complex type.